### PR TITLE
Don't crash when an executable is not found

### LINKED ISSUE #371: Not-found executable crashes framework
This was introduced with the new exe logic in 0.8.0

Here's a simple reproduction:

```yaml
- repo: local
  hooks:
  - id: test
    name: test
    language: system
    entry: i-dont-exist-lol
    files: '\.py$'
```

```
$ pre-commit run --all-files
test.....................................................................An unexpected error has occurred: OSError: Executable i-dont-exist-lol not found
Check the log at ~/.pre-commit/pre-commit.log
```

### PR DESCRIPTION
**Pull request by asottile**
Resolves #371

### CODE REVIEW

File: pre_commit/util.py [hunk 1]
```diff
@@ -163,6 +163,10 @@ def to_text(self):
         __unicode__ = to_text
 
 
+def norm_output(stdout, stderr, encoding):
+    return stdout, stderr
```
Comment (chriskuehl) [inline pre_commit/util.py]: what is this?

Comment (asottile) [inline pre_commit/util.py]: ah crap, this was my from first pass.  Will delete

**Comment by chriskuehl** [comment]
otherwise looks solid

**Pull request merged**